esperanto/malkovro/malsata.md

```diff
@@ -0,0 +1 @@
+La muro disigas en [eta estaĵoj](http://www.publicdomainpictures.net/view-image.php?image=131736&picture=halloween-ghost-cupcakes&large=1) , kriante kiel dividado . Vi eniras en [batalon](lukto/perdita.md)... 
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.